qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy.life.qbic/example/Example.groovy

```diff
@@ -0,0 +1,5 @@
+package life.qbic.example
```

Comment:
The folder structure of the project is wrong. There should not be one folder named `groovy.life.qbic` but three folders/packages `gorrvy` `life` and `qbic`

Reply:
damn, that was the IDE formatting it like this ...

Reply:
> If you want to create a number of nested directories, specify the directory names separated with slashes.